superset/config.py

```diff
@@ -63,7 +63,9 @@
 from superset.utils.encrypt import SQLAlchemyUtilsAdapter
 from superset.utils.log import DBEventLogger
 from superset.utils.logging_configurator import DefaultLoggingConfigurator
+from dotenv import load_dotenv
```

Comment:
Looks like you only need to put this in alphabetical order to make the linter happy.

Reply:
Done. Moved the line up